source/agora/utils/Log.d

```diff
@@ -54,11 +54,16 @@ public template AddLogger (string moduleName = __MODULE__)
 /// Convenience alias
 public alias LogLevel = Level;
 
-version (unittest) {}
+/// Whether logging is enabled (by default disabled in unittests)
+version (unittest)
+    __gshared bool EnableLogging = false;
 else
+    __gshared bool EnableLogging = true;
```

Comment:
So what exactly is the plan here ? Enabling logging before starting new threads ?
It also looks like you can never change it after you enabled / disabled it.

If you want to selectively enable/disable, and don't need the ability to affect other threads, you could go with something like:

```
version(unittest)
{
    void disableLogging () { Log.root.level(Level.None, true); }
    void enableLogging ()  { Log.root.level(Level.Trace, true); }
}
```

Reply:
> If you want to selectively enable/disable, and don't need the ability to affect other threads,

I do need to affect other threads. It's the child threads that run the nodes.